js_modules/dagster-ui/packages/ui-core/src/app/UserSettingsDialog/UserSettingsDialog.tsx

```diff
@@ -73,8 +79,7 @@ const UserSettingsDialogContent = ({onClose, visibleFlags}: DialogContentProps)
 
   const [arePreferencesChanged, setAreaPreferencesChanged] = React.useState(false);
 
-  const anyChange =
-    initialFlagState.current !== JSON.stringify(getFeatureFlags()) || arePreferencesChanged;
+  const anyChange = initialFlagState.current !== JSON.stringify(flags) || arePreferencesChanged;
```

Comment:
Do we need to worry about sort order on the keys for the stringified value?

Reply:
Not afaik.